Assets/HoloToolkit-Examples/HeadsUpDirectionIndicator/Example.unity

```diff
@@ -0,0 +1,431 @@
+%YAML 1.1
```

Comment:
Is this truly an example of a unit test scene? If unit test then it should reside in Utilities\Tests folder.